purchase_order_ubl/models/report.py

```diff
@@ -34,3 +34,8 @@ def get_pdf(
             pdf_content = order.embed_ubl_xml_in_pdf(
                 pdf_content)
         return pdf_content
+
+    @api.v8  # noqa
+    def get_pdf(self, records, report_name, html=None, data=None):
+        return Report.get_pdf(self._model, self._cr, self._uid, records.ids,
```

Comment:
```suggestion
        return self._model.get_pdf(self._cr, self._uid, records.ids,
```

This is a cleaner way IMO. Anyway it's not blocking.